src/morphology/english/getVerbForms.js

```diff
@@ -180,9 +180,9 @@ const endsWithIng = function( word ) {
  * @returns {boolean} True if the word ends with "ed".
  */
 const endsWithEd = function( word ) {
-	const wordLength = word.length;
-	// Consider only words of four letters or more to be past forms (otherwise, words like "red" are being treated as verb forms).
-	if ( wordLength > 3 ) {
+	const vowelCount = ( word.match( /([aeiouy])/g ) || [] ).length;
```

Comment:
Same here, you can reuse the definition of this regex at the top of the file.
Another advantage: if you define it only once, you only have to change it in one place (if it ever changes).

Reply:
Indeed!